projects/core/src/cart/facade/active-cart.service.spec.ts

```diff
@@ -291,6 +291,22 @@ describe('ActiveCartService', () => {
       expect(service['guestCartMerge']).toHaveBeenCalledWith('cartId');
     });
 
+    it('should dispatch load for current -> emulated user switch', () => {
+      service['userId'] = 'fdsfds-fsdfsd-fdsfsd-fsd';
```

Comment:
It's a pain to make sure it's the same string in the `expect` ;-)
"Ala-ma-kota" would work better her IMO :D